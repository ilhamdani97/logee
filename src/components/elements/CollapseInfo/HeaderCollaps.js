import React from 'react';
import styles from './styles.scoped.css';
import PropTypes from 'prop-types';
import { Typography } from 'logeeport-ui';

const HeaderCollaps = ({ data }) => {
  return (
    <div>
      <Typography className={styles.colapHead} type="p">
        {data.title}
      </Typography>
      <Typography className={styles.colapSubHead} type="p">
        {data.subTitle}
      </Typography>
    </div>
  );
};

HeaderCollaps.defaultProps = {
  data:{},
};

HeaderCollaps.propTypes = {
  data:PropTypes.object,
};


export default HeaderCollaps;
