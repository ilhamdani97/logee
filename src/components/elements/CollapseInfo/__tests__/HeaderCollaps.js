import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import HeaderCollaps from '../HeaderCollaps';

const data =  {
  'title': 'Armada',
  'subTitle': 'Total armada berdasarkan jenis dan kapasitasnya pada periode waktu yang dipilih.',
  'total': '24'
};
describe('src/components/elements/HeaderCollaps', () => {

  test('render', () => {
    const shallow = new ShallowRenderer();
    const tree = shallow.render(<HeaderCollaps data={data}/>);
    expect(tree).toMatchSnapshot();
  });
});
