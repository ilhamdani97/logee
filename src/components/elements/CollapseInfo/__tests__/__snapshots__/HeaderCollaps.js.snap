// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/elements/HeaderCollaps render 1`] = `
<div>
  <Typography
    spacing=""
    type="p"
  >
    Armada
  </Typography>
  <Typography
    spacing=""
    type="p"
  >
    Total armada berdasarkan jenis dan kapasitasnya pada periode waktu yang dipilih.
  </Typography>
</div>
`;
